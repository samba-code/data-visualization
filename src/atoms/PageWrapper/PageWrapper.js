import React from "react";
import styled from "styled-components";
import PropTypes from "prop-types";

const StyledDiv = styled.div`
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  width: 100%;
  overflow-x: hidden;
  background: white;
`;

const PageWrapper = ({ children }) => <StyledDiv>{children}</StyledDiv>;

PageWrapper.propTypes = {
  children: PropTypes.oneOfType([
    PropTypes.arrayOf(PropTypes.node),
    PropTypes.node,
  ]).isRequired,
};

export default PageWrapper;
