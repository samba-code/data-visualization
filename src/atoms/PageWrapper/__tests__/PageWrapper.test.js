import React from "react";
import { render } from "enzyme";

import PageWrapper from "../PageWrapper";

describe("PageWrapper", () => {
  it("should match snapshot", () => {
    const wrapper = render(<PageWrapper>Some Content</PageWrapper>);
    expect(wrapper).toMatchSnapshot();
  });
});
