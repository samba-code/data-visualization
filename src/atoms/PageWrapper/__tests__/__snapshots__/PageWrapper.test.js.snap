// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageWrapper should match snapshot 1`] = `
<div
  class="PageWrapper__StyledDiv-sc-3gvzae-0 hHrrIt"
>
  Some Content
</div>
`;
